DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,25 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
-        if (!isset($config['default_formatter'])) {
-            reset($config['formatters']);
-            $config['default_formatter'] = key($config['formatters']);
-        }
-
         if (!array_key_exists($config['default_formatter'], $config['formatters'])) {
-            throw new \InvalidArgumentException(sprintf('SonataFormatterBundle - Invalid default formatter : %s, available : %s', $config['default_formatter'], json_encode(array_keys($config['formatters']))));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
I know you are against it, and I am not trying to start a flame war here again, but AFAIK the Symfony CS says exceptions should not be split into multiple lines. Am I wrong?